### PR TITLE
Prefer custom SVG when getting stat var node hierarchy path

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: internal/server/v1/variable/ancestors.go [hunk 1]
```diff
@@ -42,6 +43,14 @@ func Ancestors(
 	for {
 		if parents, ok := cache.ParentSvg[curr]; ok {
 			curr = parents[0]
+			for _, parent := range parents {
+				parts := strings.Split(parent, "/")
+				// Prefer parent from custom import group
+				if strings.HasPrefix(parts[2], "Custom") {
+					curr = parent
+					break
+				}
```
[the remark this answers is not in the document]
Comment (pradh) [reply internal/server/v1/variable/ancestors.go:52]: If you keep it like this, then please add a an `_`, like "Custom_", for safety.

**Review by pradh — APPROVED** [review]
Thanks!

File: internal/server/v1/variable/ancestors.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (shifucun) [reply internal/server/v1/variable/ancestors.go:51]: match logic in pipeline sounds better.

**Comment by shifucun** [comment]
Thanks for review!

**Pull request merged**